### DISCUSSION TITLE
什么值得买日志报错，求解

### DISCUSSION THREAD

**Discussion by wlqb2001** [qd-today/qd#551] [Q&A]
Failed at 2/27 request,
Fail assert: {"re": "error_code\":\"4", "from": "content"} from success_asserts,
Response Error : HTTP 404: Not Found,
Request URL: https://api.smzdm.com/v1/user/lottery/draw

**Comment by MoogleXX** [comment]
同样这个问题。虽然签到成功。但是任务反馈是失败的。
